cordova/config.xml

```diff
@@ -79,6 +80,9 @@
             <string>Needs photo library access to save photos</string>
         </config-file>
     </platform>
+    <universal-links>
+      <host event="redirectToItem" name="stock-staging.goodcity.hk" scheme="https" />
```

Comment:
How this will work on prduction??

Reply:
Added production link . I thought namespace in config.xml was the reason to detect the app. Thanks.